packages/components/bolt-action-blocks/src/_action-blocks-item.scss

```
@@ -67,3 +70,14 @@
     @include bolt-margin-bottom(0);
   }
 }
+
+// Spacing prop
+@each $spacing-value in $bolt-spacing-values {
+  $spacing-value-name: nth($spacing-value, 1);
+
+  .c-bolt-action-blocks--spacing-#{$spacing-value-name} {
+    .c-bolt-action-block {
+      @include bolt-padding(#{$spacing-value-name}, stretched);
+    }
+  }
```

You're creating CSS here for all eight spacing sizes (xxsmall - xxxlarge) even though only three of them are ever used based on your schema.  Not a huge deal, but over time this leads to bloat, like what's happening with our utility classes.

A possible solution for this (not tested, and it would be good to get a blessing from @sghoweri):

```suggestion
  $supported-values: ('xsmall', 'small', 'medium');

  @if index($supported-values, $spacing-value-name) {
    .c-bolt-action-blocks--spacing-#{$spacing-value-name} {
      .c-bolt-action-block {
        @include bolt-padding(#{$spacing-value-name}, stretched);
      }
    }
  }
```

---

Tested and working.